src/components/Tool/Appeal/Modals/AddContactToAppealModal/AddContactToAppealModal.tsx

```diff
@@ -100,29 +121,54 @@ export const AddContactToAppealModal: React.FC<
           handleSubmit,
           isSubmitting,
           isValid,
-        }): ReactElement => (
-          <form onSubmit={handleSubmit} data-testid="addContactToAppealModal">
-            <DialogContent>
-              <Grid item>
-                <ContactsAutocomplete
-                  accountListId={accountListId ?? ''}
-                  value={contactIds}
-                  onChange={(contactIds) => {
-                    setFieldValue('contactIds', contactIds);
-                  }}
-                  excludeContactIds={existingContactIds}
-                />
-              </Grid>
-            </DialogContent>
-            <DialogActions>
-              <CancelButton onClick={handleClose} disabled={isSubmitting} />
+        }): ReactElement => {
+          const excludedContactNames = useMemo(() => {
+            return contactIds.reduce<string[]>((result, contactId) => {
+              const excludedContact = excludedContacts.find(
+                (excludedContact) => excludedContact.contact?.id === contactId,
+              );
+              if (excludedContact && excludedContact.contact?.name) {
+                return [...result, excludedContact.contact?.name];
+              }
+              return result;
+            }, []);
+          }, [contactIds]);
+          return (
+            <form onSubmit={handleSubmit} data-testid="addContactToAppealModal">
+              <DialogContent>
+                <Grid item>
+                  {!!excludedContactNames.length && (
+                    <Alert severity="info" data-testid="excludedContactMessage">
```

Comment:
I think this `Alert` needs some bottom padding

<img width="618" alt="Screenshot 2024-09-10 at 12 30 44 PM" src="https://github.com/user-attachments/assets/efbb535e-fdf8-4a7c-ade9-150f0d6f6303">